central/vulnerabilityrequest/datastore/internal/store/postgres/store.go

```diff
@@ -105,16 +105,16 @@ func insertIntoVulnerabilityRequests(batch *pgx.Batch, obj *storage.Vulnerabilit
 		obj.GetScope().GetImageScope().GetTag(),
 		obj.GetRequesterV2().GetId(),
 		obj.GetRequesterV2().GetName(),
-		obj.GetDeferralReq().GetExpiry().GetExpiresWhenFixed(),
 		protocompat.NilOrTime(obj.GetDeferralReq().GetExpiry().GetExpiresOn()),
+		obj.GetDeferralReq().GetExpiry().GetExpiresWhenFixed(),
 		obj.GetDeferralReq().GetExpiry().GetExpiryType(),
 		obj.GetCves().GetCves(),
 		obj.GetDeferralUpdate().GetCVEs(),
 		obj.GetFalsePositiveUpdate().GetCVEs(),
 		serialized,
 	}
 
-	finalStr := "INSERT INTO vulnerability_requests (Id, Name, TargetState, Status, Expired, Requestor_Name, CreatedAt, LastUpdated, Scope_ImageScope_Registry, Scope_ImageScope_Remote, Scope_ImageScope_Tag, RequesterV2_Id, RequesterV2_Name, DeferralReq_Expiry_ExpiresWhenFixed, DeferralReq_Expiry_ExpiresOn, DeferralReq_Expiry_ExpiryType, Cves_Cves, DeferralUpdate_CVEs, FalsePositiveUpdate_CVEs, serialized) VALUES($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18, $19, $20) ON CONFLICT(Id) DO UPDATE SET Id = EXCLUDED.Id, Name = EXCLUDED.Name, TargetState = EXCLUDED.TargetState, Status = EXCLUDED.Status, Expired = EXCLUDED.Expired, Requestor_Name = EXCLUDED.Requestor_Name, CreatedAt = EXCLUDED.CreatedAt, LastUpdated = EXCLUDED.LastUpdated, Scope_ImageScope_Registry = EXCLUDED.Scope_ImageScope_Registry, Scope_ImageScope_Remote = EXCLUDED.Scope_ImageScope_Remote, Scope_ImageScope_Tag = EXCLUDED.Scope_ImageScope_Tag, RequesterV2_Id = EXCLUDED.RequesterV2_Id, RequesterV2_Name = EXCLUDED.RequesterV2_Name, DeferralReq_Expiry_ExpiresWhenFixed = EXCLUDED.DeferralReq_Expiry_ExpiresWhenFixed, DeferralReq_Expiry_ExpiresOn = EXCLUDED.DeferralReq_Expiry_ExpiresOn, DeferralReq_Expiry_ExpiryType = EXCLUDED.DeferralReq_Expiry_ExpiryType, Cves_Cves = EXCLUDED.Cves_Cves, DeferralUpdate_CVEs = EXCLUDED.DeferralUpdate_CVEs, FalsePositiveUpdate_CVEs = EXCLUDED.FalsePositiveUpdate_CVEs, serialized = EXCLUDED.serialized"
```

Comment:
not related to this PR but @rhybrillou next time we should split this long lines to make review and diff easier